common/rpc/interceptor/telemetry.go

```diff
@@ -108,7 +108,9 @@ func (ti *TelemetryInterceptor) Intercept(
 	resp, err := handler(ctx, req)
 
 	if val, ok := metrics.ContextCounterGet(ctx, metrics.HistoryWorkflowExecutionCacheLatency); ok {
-		timerNoUserLatency.Subtract(time.Duration(val))
+		userLatencyDuration := time.Duration(val)
+		timerNoUserLatency.Subtract(userLatencyDuration)
```

Comment:
timerNoUserLatency is not being used?

Reply:
So far we want both, mostly because of averaging and alerting mechanics. We'll see further if we should remove one of these counters.
We have the same problems with task user-induced latency as well.
Let's keep it for now.